### PR TITLE
feat: wallet page

### PR DESCRIPTION
**Pull request by Space-zee**

### CODE REVIEW

**Comment by Space-zee** [comment]
Note: Need to deploy UI provider

File: frontend/app/core/walletsConnect/chainInfo.ts [hunk 1]
```diff
@@ -171,36 +190,144 @@ function deployedAddresses(deploys: any): ContractAddresses {
...
     chainInfo: localhostForked,
     contractsAddresses: deployedAddresses(localhostForkedDeploys as any),
+    tokens: {
+      nativeToken: {
+        name: 'Matic',
+        symbol: 'MATIC',
+        decimals: 18,
+        icon:'/shared/wallet/maticIcon.svg',
+        address: '0x0000000000000000000000000000000000001010',
+        oraclePriceAddress: '0xab594600376ec9fd91f8e885dadf0ce036862de0'
```
Comment (ruslanpalagin) [inline frontend/app/core/walletsConnect/chainInfo.ts:210]: let's use deploys to configure smart contracts addresses as a single place of truth

**Review by ruslanpalagin — CHANGES_REQUESTED** [review]

File: frontend/app/core/walletsConnect/chainInfo.ts (hunk 1 above)
Comment (Space-zee) [reply frontend/app/core/walletsConnect/chainInfo.ts]: +

File: contracts/deployConfig.json [hunk 2]
```diff
@@ -96,6 +156,16 @@
...
+      "nativeToken" : {
+        "address": "0x0000000000000000000000000000000000001010",
+        "oraclePriceAddress": "0xab594600376ec9fd91f8e885dadf0ce036862de0"
+      },
+      "usdt": {
+        "address": "0xbae94398c358b818635475F9b6C9a36FD2cD00bE",
+        "oraclePriceAddress": "0x0a6513e40db6eb1b165753ad52e80663aea50545"
+      }
+    },
     "ChainLinkPrice": "0x0A6513e40db6EB1b165753AD52E80663aeA50545"
```
Comment (ruslanpalagin) [inline contracts/deployConfig.json:169]: UsdtPriceOracle

**Review by ruslanpalagin — APPROVED** [review]

**Pull request merged**